src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Tcp/IdentityTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.ServiceModel;
+using System.Text;
+using Xunit;
+
+namespace Security.TransportSecurity.Tests.Tcp
+{
+    public static class IdentityTests
+    {
+        [Fact]
+        [ActiveIssue(12)]
+        [OuterLoop]
+        //Once this feature is enabled, we will need to add negative tests
```

Comment:
NIT: consider adding this comment to the issue itself rather than putting it in code here.